components/file_system/src/lib.rs

```diff
@@ -266,13 +272,33 @@ impl<R: Read> Read for Sha256Reader<R> {
     }
 }
 
+const SPACE_PLACEHOLDER_FILE: &str = "space_placeholder_file";
+
+// create a file with hole, to reserve space for TiKV.
+pub fn reserve_space_for_recover<P: AsRef<Path>>(data_dir: P, file_size: u64) -> io::Result<()> {
```

Comment:
why move to this crate

Reply:
So tikv_util doesn't depend on file_system anymore.

Reply:
seems there is a lot of `std::fs` need to be replaced? it depends on file_system anyway

Reply:
What do you mean? file_system now depends on tikv_util to use the time_util, therefore tikv_util has some uses of std::fs that can't be pruned.